app/locales/en/translations.js

```diff
@@ -146,7 +146,7 @@ const translations = {
         "title": "Your Personality"
     },
     "flag": {
-        "chooseLanguage": "Please select your country and language"
+        "chooseLanguage": "Please choose a language"
```

Comment:
Note to self: this language change will need to be reverted. It was made at the request of the client, and apparently they did not update their source spreadsheet.